### ISSUE TITLE
ERNIE-GEN returns multiple inference models

### ISSUE THREAD

**Issue by JinghongZeng** [PaddlePaddle/ERNIE#902]
I am using ERNIE-GEN to do text generation as shown in [here](https://github.com/PaddlePaddle/ERNIE/tree/ernie-kit-open-v1.0/applications/tasks/text_generation). After training ERNIE, the folder `save_inference_model` contains multiple inference models. In the training process, every 10 steps a model is saved. So the folder `save_inference_model` contains inference models every 10 steps up to 260 steps, and also a 261-step model which is shown as the final evaluation from the modelling console. I found that the predictions from inference models of 250, 260, 261 steps are the same. Does this mean that the final-step inference model is the final model for inference?

Screenshot of the folder `save_inference_model`:
<img width="997" alt="Screenshot 2023-04-18 at 10 19 27" src="https://user-images.githubusercontent.com/86920951/232653404-88585cbd-13b8-481e-902a-3b50963d239c.png">

Steps 259-261 output from console:
`INFO: 04-18 06:03:34: custom_static_generation_trainer.py:79 * 8611209856 epoch 19 progress 99/100 pyreader queue size 0 learning_rate 0.0
INFO: 04-18 06:03:34: ernie_infilling_generation.py:484 * 8611209856 step: 259, loss: 2.174187, ppl: 8.795, speed: 0.015 steps/s
INFO: 04-18 06:03:34: ernie_gen_infilling_dataset_reader.py:156 * 8611209856 set data_generator and start.......
INFO: 04-18 06:04:16: ernie_infilling_generation.py:508 * 8611209856 [259_evaluate evaluation] bleu: 0.282412, elapsed time: 34.381865 s
INFO: 04-18 06:04:36: custom_static_generation_trainer.py:79 * 8611209856 epoch 19 progress 99/100 pyreader queue size 0 learning_rate 0.0
INFO: 04-18 06:04:36: ernie_infilling_generation.py:484 * 8611209856 step: 260, loss: 2.275375, ppl: 9.732, speed: 0.016 steps/s
INFO: 04-18 06:04:36: ernie_gen_infilling_dataset_reader.py:156 * 8611209856 set data_generator and start.......
INFO: 04-18 06:05:15: ernie_infilling_generation.py:508 * 8611209856 [260_evaluate evaluation] bleu: 0.282412, elapsed time: 33.458970 s
INFO: 04-18 06:05:15: static_trainer_ernie_gen.py:470 * 8611209856 save model on static....
INFO: 04-18 06:05:27: custom_static_generation_trainer.py:106 * 8611209856 Final evaluate result: 
INFO: 04-18 06:05:27: ernie_gen_infilling_dataset_reader.py:156 * 8611209856 set data_generator and start.......
INFO: 04-18 06:06:12: ernie_infilling_generation.py:508 * 8611209856 [261_evaluate evaluation] bleu: 0.282412, elapsed time: 37.477112 s
INFO: 04-18 06:06:12: static_trainer_ernie_gen.py:470 * 8611209856 save model on static....
INFO: 04-18 06:06:24: <stdin>:15 * 8611209856 end of run train and eval .....`
